components/atoms/Icon/Icon.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import tailwindConfig from '../../../tailwind.config.js'
+import {icons} from './icons'
+
+/**
```

Comment:
I do wonder if  `sizeToPx()` could be abstracted to a function file, since it appears to be something that might be re-useable in the future. What are your thoughts?

Reply:
We should normally write it in REM but for SVG width and height attributes, only px are valid.

https://bugzilla.mozilla.org/show_bug.cgi?id=1231147